test/core/simd/meta/simd_f64x2.py

```diff
@@ -0,0 +1,443 @@
+#!/usr/bin/env python3
+
+"""
+Generate f64x2 [abs, min, max] cases.
+"""
+
+from simd_f32x4 import Simdf32x4Case
+from simd_f32x4_arith import Simdf32x4ArithmeticCase
+from test_assert import AssertReturn
+
+
+class Simdf64x2Case(Simdf32x4Case):
+
+    LANE_TYPE = 'f64x2'
+
+    FLOAT_NUMBERS = (
+        '0x0p+0', '-0x0p+0', '0x1p-1074', '-0x1p-1074', '0x1p-1022', '-0x1p-1022', '0x1p-1', '-0x1p-1', '0x1p+0', '-0x1p+0',
+        '0x1.921fb54442d18p+2', '-0x1.921fb54442d18p+2', '0x1.fffffffffffffp+1023', '-0x1.fffffffffffffp+1023', 'inf', '-inf'
+    )
+
+    NAN_NUMBERS = ('nan', '-nan', 'nan:0x4000000000000', '-nan:0x4000000000000')
+
+    def gen_test_fn_template(self):
+
+        # Get function code
+        template = Simdf32x4ArithmeticCase.gen_test_fn_template(self)
+
+        # Function template
+        tpl_func = '  (func (export "{}"){} (result v128) ({} {}{}))'
```

Comment:
What does the `tpl` here mean? It also might be good to use names in the string interpolation so its clear what each part is supposed to be.